Automattic/wp-calypso
# [e2e] Check signup processing screen

While running Signup tests locally, very often, I see `Signup Processing Page was not displayed or moved too quickly for the test` message in the log during `Can then see the sign up processing page which will finish automatically move along` step. I changed the way how we are checking is selector `signup-processing-screen__content` present. 

#### Testing instructions

Run some Signup test locally, for example, `Sign up for a free WordPress.com site from the Jetpack new site page, and log in via a magic link` and make sure that you don't see that message.
